micro-rdk/src/common/conn/viam.rs

```diff
@@ -622,6 +600,42 @@ where
             }
         }
 
+        #[cfg(feature = "ota")]
```

Comment:
moved the whole ota section to after certs have been retrieved and stored

Reply:
Why do we need certs? I thought the certs were for us _serving_

Reply:
oh, I am using the same certs for the hyper client's https connection.  should I not be?

Reply:
Let's try the empirical answer: does it still work if you don't use them?

Reply:
you were right. we just need the certificate authority, which the H2 connector is already using. removed the certificates completely.